Dockerfile

```diff
@@ -0,0 +1,14 @@
+# This Dockerfile builds a mainnet version for API3 DAO dashboard
+
+FROM node:lts-alpine as builder
+RUN apk add --no-cache git
+WORKDIR /usr/src/app
+ADD . .
+COPY src/contract-deployments/mainnet-dao.json ./src/contract-deployments/localhost-dao.example.json
```

Comment:
You actually don't need this line - it will be done as part of `yarn postinstall`

Reply:
nope, post install will copy your local setup. There is a trick of how to connect to mainnet. You need to study source